examples/virus_on_network_directed/README.md

```diff
@@ -0,0 +1,46 @@
+# Virus on a Network
+
+## Summary
+
+This model is based on the NetLogo model "Virus on Network".
```

Comment:
Needs to say that this model is almost the same as examples/virus_on_network except that the graph is directed. I haven't fully read the code to confirm that this is the only difference.

Reply:
I've added some additional text to clarify that the only difference is that the graph is directed.